src/routes/Profile/ProfileFront/ProfileFront.tsx

```diff
@@ -0,0 +1,65 @@
+import { components } from '@/api/schema';
+import { MainLayout } from '@/components';
+import { ExternalLink } from '@/components/ExternalLink/ExternalLink';
+import { ProfileNavigation } from '@/components/MainLayout/ProfileNavigation';
+import React from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { useRouteLoaderData } from 'react-router';
+
+const ProfileFront = () => {
+  const {
+    t,
+    i18n: { language },
+  } = useTranslation();
+
+  const { etunimi } = useRouteLoaderData('root') as components['schemas']['OhjaajaCsrfDto'];
+
+  const navChildren = React.useMemo(() => <ProfileNavigation />, []);
+
+  return (
+    <MainLayout navChildren={navChildren}>
+      <title>{t('profile.front.title')}</title>
+      <h1 className="mb-6 text-heading-2 sm:text-heading-1" data-testid="profile-front-title">
+        {t('profile.front.greeting', {
+          etunimi,
+        })}
+      </h1>
+
+      <div className="mb-8 text-body-md flex flex-col gap-7">
+        <p className="text-body-lg">{t('profile.front.you-are-signed-in')}</p>
+        <div className="font-arial">
+          <p>{t('profile.front.profile-description')}</p>
+          <ul>
+            <li className="list-disc ml-9 pl-1">{t('profile.front.list-1-item-1')}</li>
```

Comment:
Could these `<li>` use same small component in this file, as they all have same `className`s?